src/Markdig/Helpers/CharHelper.cs

```diff
@@ -1,6 +1,27 @@
-﻿// Copyright (c) Alexandre Mutel. All rights reserved.
```

Comment:
You should not remove the original license but append the .NET MIT License after (with a one line preamble that explains which code was taken from .NET Core, ,that should be enough)